docs/index.adoc

```diff
@@ -394,25 +394,87 @@ creation of a new active wallet regardless of age.
 |Yes
 |`1 week`
 
-|movedFundsSweepTimeoutSlashingAmount
-|The amount of stake to slash if the wallet does not move its funds in time.
-|Yes
-|`100% of min-stake`
-
 |movedFundsSweepTimeoutNotifierRewardMultiplier
 |The the percentage of the slashed stake that the notifier receives as a reward.
 |Yes
 |`5%`
 
 |walletMaxBtcTransfer
-|The threshold at which we try to divide up a closing wallet into multiple target wallets
+|The amount of satoshis at which we try to divide up a closing wallet into multiple target wallets.
 |Yes
-|`50 BTC`
+|`10e8`
 
 |walletClosingPeriod
 |The amount of time the wallet remains in the `Closing` state before it is closed.
 |Yes
-|`3 days`
+|`40 days`
+
+|walletClosureMinBtcBalance
+|The minimum amount of satoshis at which a wallet is eligible to begin closing.
+|Yes
+|`5 * 1e7`
+
+|walletMaxAge
+|The age at which a wallet is eligible to begin closing.
+|Yes
+|`26 weeks`
+
+|movingFundsTxMaxTotalFee
+|The max amount of satoshis that the wallet is allowed to pay miners for a moving funds transaction.
+|Yes
+|`10000 satoshis`
+
+|movingFundsDustThreshold
+|The minimum amount of satoshis that we move from wallet to wallet when closing a wallet.
```

Comment:
I would try to avoid using `we` when we speak about wallet's activities.
Maybe
```suggestion
|The minimum amount of satoshis held by a closing wallet that needs to be moved to a different wallet before closing.
```
?

Reply:
[54b7a3a](https://github.com/keep-network/tbtc-v2/pull/346/commits/54b7a3aa103d9dfe4e3cce4542eb7b703710ba88)